releasenotes/notes/add_flag_for_span_aggregator_rlock-b08c88e0da85dc66.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    sqlalchemy: This fix resolves an issue where sqlalchemy rollbacks could intermittently cause deadlocks.
```

Comment:
We might want to have this as a known issue instead of a fix?

Since this isn't introducing the fix, this is introducing an opt-in to the possible fix for customer validation. When we change to make the RLock the default, then we can introduce it as a fix?

cc @mabdinur who has been thinking about introducing "known issues" for a bit now.

Reply:
Yup a known issue release note + mitigation is what we need here.

Reply:
Hmmmm.... should "known issues" just be "issues"? Since we already have that as an option, the release.py script won't be that happy adding "known issues", given it likes to use regex to parse release notes, and the fact that if we're writing about an issue, it implies that it is known?